cmake/Modules/FindAPL.cmake

```diff
@@ -0,0 +1,50 @@
+SET(APL_INCLUDE_SEARCH_PATHS
+$ENV{ARMPL_DIR}/include
+)
+
+SET(APL_LIB_SEARCH_PATHS
+$ENV{ARMPL_DIR}/lib
+ )
+
+FIND_PATH(APL_INCLUDE_DIR NAMES armpl.h PATHS ${APL_INCLUDE_SEARCH_PATHS})
+FIND_LIBRARY(APL_LIB NAMES armpl_lp64.dll.lib PATHS ${APL_LIB_SEARCH_PATHS})
+FIND_LIBRARY(APL_MATH NAMES amath.dll.lib PATHS ${APL_LIB_SEARCH_PATHS})
```

Comment:
Aren't APL also available for Linux? Is there a standard Cmake module available one can copy-n-pasted?

Reply:
Upstream CMake's version of FindBLAS - https://cmake.org/cmake/help/latest/module/FindBLAS.html

Reply:
Hey @malfet, I want to better understand the requirements. Is it okay to just update the FindAPL module to support linux too? @nSircombe thanks for the links, but it looks like the FindBlas/find_package imlementations do not recognize ArmPL on Windows currently: https://community.arm.com/support-forums/f/high-performance-computing-forum/56375/find_package-blas-fails-to-find-arm-performance-libraries/184602